rpcs3/Emu/Cell/Modules/cellSysmodule.cpp

```diff
@@ -120,6 +120,7 @@ static const char* get_module_name(u16 id)
 	case 0xf030: return "cellSysutilAvc2";
 	case 0xf034: return "cellSailRec";
 	case 0xf035: return "sceNpTrophy";
+	case 0xf044: return "cellSysutilNpEula";
```

Comment:
@Zangetsu38 nope, the correct name for the module is that one. If you want to change it you'd have to ask sony engineers.